src/openapi.test.ts

```diff
@@ -134,7 +135,7 @@ describe('toOpenAPISpec', () => {
             ],
             responses: {
               '200': {
-                description: 'This endpoint has a description',
+                description: 'A successful response',
```

Comment:
Is this overkill here?

Reply:
The field is weirdly required in the OpenAPI spec. If we don't include it, the spec won't pass validators.